Cargo.toml

```diff
@@ -43,7 +43,7 @@ sea-query-attr = { version = "^0.1.1", path = "sea-query-attr", optional = true
 sea-query-derive = { version = "0.2.0", path = "sea-query-derive", optional = true }
 sea-query-driver = { version = "^0.1.1", path = "sea-query-driver", optional = true }
 serde_json = { version = "^1", optional = true }
-bytes = { version = "^1", optional = true }
+bytes = { version = "^1.1", optional = true }
```

Comment:
Why this change?

Reply:
For some reason, since the new version of sqlx requires 1.1, it resolved bytes as 1.0 in some crates and then it was incompatible with the 1.1 required by sqlx. That's the easiest way I found to make it work.